src/test/scala/codacy/docker/api/utils/TimeoutSpecs.scala

```diff
@@ -0,0 +1,27 @@
+package codacy.docker.api.utils
+
+import org.specs2.mutable.Specification
+
+import scala.concurrent.duration._
+import scala.concurrent.{Await, TimeoutException}
+
+class TimeoutSpecs extends Specification with Timeout {
+
+  "Timeout" should {
+    "should throw exception" in {
+      val f = initTimeout(100.seconds) {
+        failure("It should fail because a timeoutException wasn't thrown.")
```

Comment:
Change the text to "A scala.concurrent.TimeoutException" was expected to make the test easier to understand.